chia/wallet/util/wallet_types.py

```diff
@@ -32,8 +32,13 @@ class CoinType(IntEnum):
     CLAWBACK = 1
 
 
-class RemarkDataType(IntEnum):
-    CLAWBACK = -991
+class RemarkDataType:
+    """
+    We use a relative random unique number here to represent a type of Remark announcement.
+    This will reduce the chance of collision with Remark announcements in other puzzles.
+    """
+
+    CLAWBACK = bytes.fromhex("C21B5973")
```

Comment:
It's just a random number. Actually, I'm not sure how many digits are good enough. Any suggestion on digits and number?

Reply:
I think the `CLAWBACK` string is pretty good. to save bytes I suppose you could make it shorter, like `CLWBK`.
I don't have a strong opinion though, it might be good to leave a comment explaining that it's used as the `REMARK` condition to identify these, and that it should be sufficiently unique

Reply:
Why do we need more than 1 or 2 bytes here? Wouldn't it be sufficient to just start a `uint8` mapping here considered we probably never will en up with 256 different remark data types? Or if there is a concern about it maybe `uint16` and then just based on the first n bytes try to deserialize the remaining data into the type we map it to and if it passes, its all fine and if the deserialization fails its something we don't care about. At the end even if we use `CLAWBACK` or `CLWBK` anyone can create remark data with this prefix and invalid remaining data so we anyway need to consider it. Or do i miss something important here now?

Reply:
The only downside of using a common number is we may see some false negative error logs.